Makefile

```diff
@@ -47,25 +48,10 @@ endif
 .PHONY: setup
 setup:
 	$(call LOG,Starting Flyte sandbox)
-	docker run -d --rm --privileged --name $(FLYTE_SANDBOX_NAME) \
-		-e SANDBOX=1 \
-		-e KUBERNETES_API_PORT=$(KUBERNETES_API_PORT) \
-		-e FLYTE_HOST=localhost:30081 \
-		-e FLYTE_AWS_ENDPOINT=http://localhost:30084/ \
-		-v $(CURDIR):/usr/src \
-		-v $(KUBE_CONFIG):/etc/rancher/ \
-		-p $(KUBERNETES_API_PORT):$(KUBERNETES_API_PORT) \
-		-p $(FLYTE_PROXY_PORT):30081 \
-		-p $(K8S_DASHBOARD_PROXY_PORT):30082 \
-		-p $(MINIO_PROXY_PORT):30084 \
-		cr.flyte.org/flyteorg/flyte-sandbox:dind > /dev/null
-
-.PHONY: wait
-wait:
-	$(call RUN_IN_SANDBOX, wait-for-flyte.sh)
+	flytectl sandbox start --flytesnacks=$(shell pwd)
```

Comment:
What does `--flytesnacks` do?

Reply:
it will copy the flytesnack repo inside the sandbox container that is the requirement for running everything in the container

Reply:
ic...

Reply:
so we need to update all `--flytesnacks` to `--sourcePath`